transitions/registration.js

```diff
@@ -265,6 +265,7 @@ module.exports = {
             if (err) {
                 callback(err);
             } else if (registrations.length) { // id collision, retry
+                logger.warn('Registration ID ' + id + ' is not unique, retrying...');
```

Comment:
Yeah there is an issue with more details and probably some ideas for better solutions somewhere, I can dig it up...